dask_conversions/done/Dask_Coastline_Extraction_Landsat.ipynb

```diff
@@ -23,12 +23,10 @@
     "\n",
```

Comment:
<p>I don't think dask is required for this part as it is just changing column headings to date times. Seems like you have it working! I wonder if it is here where we are dropping data from 2012 to 2019. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SatelliteApplicationsCatapult/odc-hub/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Please add to this cell </p><p>############</p><p>elif 'tide' in df.columns:</p><p>&nbsp;&nbsp;&nbsp;&nbsp;df['tides'] = df['tide']</p><p>&nbsp;&nbsp;&nbsp;&nbsp;df = pd.DataFrame.drop(df, columns=['tide'])</p><p>###############</p><p>this was causing some data to be missed later on in the workflow due to naming being different between column headings</p>